doc/cookbook/source/examples/distance/cosine.rst

```diff
@@ -0,0 +1,39 @@
+==================
+Cosine	 Distance
+==================
+
+The Cosine distance for real valued features is the similarity between two vectors by finding their angle.
+
+.. math::
+
+    1-\frac{{\bf x^\top x'}}{\sqrt{\sum_{i=1}^{d}{\bf {x_i}^2}\sum_{i=1}^{d}{\bf {x'_i}^2}}}
```

Comment:
I just rendered the math here. It needs some minor tweaks (sorry for not checking earlier)
the _i subscript should not be in the bf macro, i.e. \bf {x}_i

Can you make write the denominator as `\Vert \bf{x}\Vert_2 \Vert \bf{x'}\Vert_2` and then say " where :math:`\Vert \cdot\Vert_2` is the Euclidean norm.